lib/src/ovis_auth/auth.c

```diff
@@ -65,6 +65,7 @@
 #include <stdio.h>
 #include <arpa/inet.h>
 #include "auth.h"
+#include "ovis_util/util.h"
```

Comment:
Why include ovis_util/util.h as part of this commit?

Reply:
This was the start of getting rid of the strerror_r/strerror from auth, and then I paused because it adds util to the link line of auth.  In the interest of getting rid of all strerror usage, I'll go ahead and finish adding util to the link.